lib/class/xml_data.class.php

```diff
@@ -480,6 +480,14 @@ public static function indexes($objects, $object_type, $full_xml = true, $includ
                 case 'video':
                     $string .= self::videos($objects);
                     break;
+                case 'live_stream':
+                    $live_stream = new Live_Stream($object_id);
+                    $live_stream->format();
```

Comment:
don't think the format is needed as f_name is just scrubbed just finalising now.